### PR TITLE
Add compatibility for Symfony 6

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #issuenum
| Related issues/PRs | #issuenum
| License | MIT

#### What's in this PR?

Add compatibility for Symfony 6.

#### Why?

Compatiblity to Symfony 6 and get ride of deprecated swiftmailer.

### CODE REVIEW

**Review by luca-rath — APPROVED** [review]

File: Tests/Application/config/swiftmailer.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+swiftmailer:
+    disable_delivery:  true
```
Comment (luca-rath) [inline Tests/Application/config/swiftmailer.yml:2]: One question, why do we have a swiftmailer config, if swiftmailer isn't installed?

Comment (alexander-schranz) [reply Tests/Application/config/swiftmailer.yml:2]: its installed when using Symfony 5.4 or Sulu 2.4 (lower dependencies)

Comment (alexander-schranz) [reply Tests/Application/config/swiftmailer.yml:2]: and there its still the default mailer to not break projects which maybe did create swiftmailer listeners.

Comment (alexander-schranz) [reply Tests/Application/config/swiftmailer.yml:2]: handled here: https://github.com/sulu/SuluFormBundle/blob/ba31c3e69d477b5665e24ffb23d91bbd446e6edc/DependencyInjection/SuluFormExtension.php#L244-L258

**Comment by rogamoore** [comment]
Would it be possible to release these changes? I noticed that currently our projects are unable to update to Sulu 2.5 because it fails with

```
!!    Compile Error: Declaration of Sulu\Bundle\FormBundle\Controller\FormWebsite
!!    Controller::getSubscribedServices() must be compatible with Sulu\Bundle\Web
!!    siteBundle\Controller\WebsiteController::getSubscribedServices(): array
```

From the changes here I guess that should be fixed already. :)

**Comment by alexander-schranz** [comment]
@rogamoore we want to first test the demo with all upgraded bundles before doing a stable release of the bundles. If you already want to use you can install this via using `^2.5@dev`  in your composer.json for the sulu/form-bundle.

**Pull request merged**